ci-operator/step-registry/ipi/conf/mirror/ipi-conf-mirror-commands.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+mkdir -p ${HOME}/.docker
+cp ${CLUSTER_PROFILE_DIR}/pull-secret ${HOME}/.docker/config.json
+oc registry login
+MIRROR_BASE=$( oc get is release -o 'jsonpath={.status.publicDockerImageRepository}' )
+oc adm release new \
+  --from-release ${RELEASE_IMAGE_LATEST} \
+  --to-image ${MIRROR_BASE}-scratch:release \
+  --mirror ${MIRROR_BASE}-scratch \
+  || echo 'ignore: the release could not be reproduced from its inputs'
```

Comment:
This came in with 246118f38e897fa6027202a8d26c9ac07060cee4 (#4585).  I forget why we were seeing that error.  Might have been different Go versions generating different gzip layers?  Would be nice if we had a more informative error message, or an option to ignore this specific failure mode, or some such, so we didn't have to blindly ignore the exit code here.  Thoughts?